SofaKernel/modules/SofaBaseCollision/src/SofaBaseCollision/ContactListener.cpp

```diff
@@ -113,5 +114,63 @@ void ContactListener::handleEvent( core::objectmodel::Event* _event )
     }
 }
 
+unsigned int ContactListener::getNumberOfContacts() const {
```

Comment:
(Guidelines)
- It would be preferable to use the new sofa::Size type (which is just a unsigned int at the end), defined sofa/config.h
- curly brackets should be on a new line.
E.g
```
sofa::Size ContactListener::getNumberOfContacts() const
{
...
}
```